src/Controllers/Version2/class-wc-rest-system-status-v2-controller.php

```diff
@@ -571,11 +566,12 @@ public function get_item_schema() {
 	/**
 	 * Return an array of sections and the data associated with each.
 	 *
+	 * @param array $fields List of fields to be included on the response.
 	 * @return array
 	 */
-	public function get_item_mappings() {
+	public function get_item_mappings( $fields ) {
```

Comment:
> Should it default to all fields?

Doing it makes impossible to avoid a expensive call every time that just need a single request just to fetch the version.